### ISSUE TITLE
[Bug Report][3.3.23] <v-spacer> lost "flex-grow: 1" attitube after build

### ISSUE THREAD

**Issue by ailtariel-lin** [vuetifyjs/vuetify#18924]
### Environment
**Vuetify Version:** 3.3.23
**Vue Version:** 3.3.4
**Vite Version:** 4.4.5
**Browsers:** Firefox 122.0
**OS:** Windows 10

### tsconfig.json
```
{
  "compilerOptions": {
    "target": "ES2020",
    "useDefineForClassFields": true,
    "module": "ESNext",
    "lib": ["ES2020", "DOM", "DOM.Iterable"],
    "skipLibCheck": true,

    /* Bundler mode */
    "moduleResolution": "bundler",
    "allowImportingTsExtensions": true,
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "preserve",

    /* Linting */
    "strict": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noFallthroughCasesInSwitch": true,

    // custom
    "esModuleInterop": true,
    "noImplicitAny": false,
    "paths": {
      "@/*": ["./src/*"]
    }
  },
  "include": ["src/**/*.ts", "src/**/*.d.ts", "src/**/*.tsx", "src/**/*.vue"],
  "references": [{ "path": "./tsconfig.node.json" }]
}
```

### Steps to reproduce
It works fine in development, but after build, <v-spacer> will lose "flex-grow: 1" attitube. Maybe only in <v-app-bar> or <v-toolbar>, i'm not sure.

### Expected Behavior
same performance before and after build

### Actual Behavior
not the same

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpdkMFqwzAQRH9lq4tbqO1CcyjFKfTWP+ih7kFWlCKQtMtqEygh/x5JDkmc287TMNrZn4NKbPpPom6/s+pdDWIDeS32Y4wAw77VRHWswmAU7aLlM6pQEP2kr6jCRNpk29BfxsXzJBEMeuT1qIhd0Pw/KnA5/8ZXnZVB2LjWeEwWSuKdr6AceN0p6+VShSx3L2SuNvQ3lbNMhh0JJCu7ubkLhCxwALZbOMKWMUCTr9UUO+QaMQmE9Afr4nhsvqz3CN/IfvPQPJUP5sgcpo7P6rVbdS/tZEV3KzXLN/V7AtgYehE=)

**Comment by websitevirtuoso** [comment]
To help you, you need to provide repo with you configuration, only after that somebody will check it.

Right now it works for many people in production

**Comment by websitevirtuoso** [comment]
Do you still need help with it? if yes please repo or issue will be closed after some time

**Comment by KaelWD** [comment]
Duplicate of #18121
